pkg/cmd/version/version.go

```diff
@@ -52,20 +53,41 @@ func Command(p cli.Params) *cobra.Command {
 			"commandType": "utility",
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+			if component == "" {
+				fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+			}
 
 			cs, err := p.Clients()
 			if err == nil {
-				pipelineVersion, _ := version.GetPipelineVersion(cs)
-				if pipelineVersion == "" {
-					pipelineVersion = "unknown"
-				}
-				fmt.Fprintf(cmd.OutOrStdout(), "Pipeline version: %s\n", pipelineVersion)
-				triggersVersion, _ := version.GetTriggerVersion(cs)
-				if triggersVersion == "" {
-					triggersVersion = "unknown"
+				switch component {
+				case "":
+					pipelineVersion, _ := version.GetPipelineVersion(cs)
+					if pipelineVersion == "" {
+						pipelineVersion = "unknown"
+					}
+					fmt.Fprintf(cmd.OutOrStdout(), "Pipeline version: %s\n", pipelineVersion)
+					triggersVersion, _ := version.GetTriggerVersion(cs)
+					if triggersVersion == "" {
+						triggersVersion = "unknown"
+					}
+					fmt.Fprintf(cmd.OutOrStdout(), "Triggers version: %s\n", triggersVersion)
```

Comment:
You can probably capture this as a function instead and just pass in the component value instead of using a switch statement. Some like `func displayVersions(component string) error`.